wandb/integration/magic.py

```diff
@@ -484,6 +485,9 @@ def magic_install(init_args=None):
     if _run_once:
         return
     _run_once = True
+    if get_optional_module("tensorflow") is None:
```

Comment:
I think we should move this code block to be above the instantiation of the global _run_once variable.